common/src/main/java/io/netty/util/internal/InternalThreadLocalMap.java

```diff
@@ -164,13 +164,17 @@ public int size() {
     }
 
     public StringBuilder stringBuilder() {
-        StringBuilder builder = stringBuilder;
-        if (builder == null || builder.capacity() > STRING_BUILDER_MAX_CAPACITY) {
```

Comment:
maybe just upper this to 1024 (as it is per thread anyway) and then its good enough for now. we can add a property later if we need.

@jasontedor WDYT ?